From cf7e328c931a46997fbc4a40540afccd2da8f1e9 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Thu, 9 Sep 2021 13:00:36 +0100
Subject: [PATCH] docs: Don't scan ctx.h with gtk-doc

ctx.h contains both declarations and implementation, causing gtk-doc to
fail. It isn't part of GEGL's public API anyway, so don't try to
document it as such.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 docs/reference/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/reference/meson.build b/docs/reference/meson.build
index 8193a25ba..2348cec92 100644
--- a/docs/reference/meson.build
+++ b/docs/reference/meson.build
@@ -12,6 +12,7 @@ if gtkdoc_scan.found()
       project_source_root / 'gegl',
       project_build_root / 'gegl',
     ],
+    ignore_headers: ['ctx.h'],
     fixxref_args: [
       '--html-dir=@0@'.format(gegl_gtk_docdir),
       '--extra-dir=@0@'.format(glib_gtk_docdir),
-- 
GitLab


